specs/lib/mergeDuplicateMaterialsSpec.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+var mergeDuplicateMaterials = require('../../lib/mergeDuplicateMaterials');
+
+describe('mergeDuplicateMaterials', function() {
+    it('merges duplicate materials', function() {
+        var gltf = {
+            meshes : {
+                meshOne : {
+                    primitives : [
+                        {
+                            material : 'materialOne',
+                        },
+                        {
+                            material : 'materialThree'
+                        }
+                    ]
+                },
+                meshTwo : {
+                    primitives : [
+                        {
+                            material : 'materialTwo'
+                        },
+                        {
+                            material : 'materialThree'
+                        }
+                    ]
+                }
+            },
+            materials : {
```

Comment:
Yes, but see https://github.com/AnalyticalGraphicsInc/gltf-pipeline/pull/152/files/1d35880f23bafa7c6e685e592077c11b5c41ca50#r73891779.

We would still keep the unit tests for removing; this would be more of an integration test since removing is still part of the function's contract.